packages/base/src/UI5Element.js

```diff
@@ -590,6 +590,19 @@ class UI5Element extends HTMLElement {
 		return this[slotName].reduce(reducer, []);
 	}
 
+	get isCompact() {
+		return getComputedStyle(this).getPropertyValue("--_ui5_content_density") === "compact";
+	}
+
+	updateStaticAreaItemContentDensity() {
+		const staticAreaItem = this.getStaticAreaItemDomRef().host;
```

Comment:
IMO it's a bit safer to check for ".host" only if there is staticAreaItemDomRef()